src/ecalc/libraries/libecalc/common/libecalc/core/graph_result.py

```diff
@@ -119,12 +119,7 @@ def _evaluate_installations(self, variables_map: dto.VariablesMap) -> List[libec
                 ],
             )
 
-            aggregated_emissions = aggregate_emissions(
-                [
-                    self.emission_results[fuel_consumer_id]
-                    for fuel_consumer_id in self.graph.get_successors(installation.id)
-                ]
-            )
+            aggregated_emissions = aggregate_emissions(self.emission_results.values())
```

Comment:
Godt poeng, må sjekke litt

Reply:
Av det jeg kan se gir gammel og ny kode identisk resultat. Testet på ltp-eksporten (flere installasjoner). Det er det at en looper over installasjoner, og at aggregated emissions til en hver tid inneholder emissions for en gitt installasjon. Ergo så jeg ikke helt behovet for den gamle koden. Hvis alle emissions tas med for hver installasjon burde det vel vært fanget opp i testene som har flere installasjoner?

Reply:
Ja, graph-objektet inneholder flere installasjoner. Ellers, testen for ltp-eksport inneholder emissions>0 for alle installasjoner (flere typer utslipp)

Reply:
k. I do not understand that this will work. It seems to be that the curent code is incorect and will aggregate across all instalaltions for each installation. @jorgenengelsen @olelod can you please help out to verify what is correct?

Reply:
Jeg er enig, det må endres tilbake. Ser det nå, dette vil aggregere over alle installasjoner hver gang. Når jeg endrer tilbake får jeg dog ingen endring på testene (resultat). Lurer på om testene ikke fanger opp akkurat dette.

Reply:
@TeeeJay, har fått in test i test_aggregators.py

Reply:
nice! og da fikk du bekrefta at den alternative koden ble feil?

Reply:
Jepp